### PR TITLE
refactor: use nullish coalescing

### PR DESCRIPTION
**Pull request by marcoscaceres**

### CODE REVIEW

**Comment by marcoscaceres** [comment]
Test failing because of #3626

File: src/core/xref.js [hunk 1]
```diff
@@ -132,7 +132,7 @@ function normalizeConfig(xref) {
       if (xref.profile) {
         const profile = xref.profile.toLowerCase();
         if (profile in profiles) {
-          const specs = (xref.specs || []).concat(profiles[profile]);
+          const specs = xref.specs?.concat(profiles[profile]) ?? [];
```
Comment (sidvishnoi) [inline src/core/xref.js:135]: The result is not the same here.

Comment (sidvishnoi) [inline src/core/xref.js:135]: The result is not the same here.

**Review by sidvishnoi — CHANGES_REQUESTED** [review]

File: src/core/xref.js (hunk 1 above)
Comment (marcoscaceres) [reply src/core/xref.js:135]: ah, right.

**Review by sidvishnoi — APPROVED** [review]
Would love to find more potential uses and create a larger PR 😄

**Pull request merged**